scripts/Microsoft.DotNet.Cli.Build.Framework/TargetConditions/EnvironmentAttribute.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Cli.Build.Framework
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true, Inherited = false)]
+    public class EnvironmentAttribute : TargetConditionAttribute
+    {
+        private string _envVar;
+        private string _expectedVal;
+
+        public EnvironmentAttribute(string envVar, string expectedVal)
+        {
+            if (string.IsNullOrEmpty(envVar))
+            {
+                throw new ArgumentNullException("envVar");
+            }
+
+            _envVar = envVar;
+            _expectedVal = expectedVal;
+        }
+
+        public override bool EvaluateCondition()
+        {
+            var actualVal = Environment.GetEnvironmentVariable(_envVar);
+
+            if (string.IsNullOrEmpty(_expectedVal))
+            {
+                return string.IsNullOrEmpty(actualVal) ||
```

Comment:
Hmm.. thinking about it I want to change the Environment Attribute itself to 
`[Environment(string envName, params string[] values)]`. And the `TargetCondition` succeeds if the environment variable has any of the `values`. This way I can use this attribute like below
```
[Environment("SKIP_PACKAGE", null, "0", "false")]
public static BuildContext PackageTarget(){
}
```